2021 03 02 iswilli accessreviewscopeupdates (#11842)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/accessreviewqueryscope.md

---
title: "accessReviewQueryScope resource type"

> description: "accessReviewQueryScope defines what will be reviewed in an access review"

Reviewer: ```suggestion
description: "Defines what will be reviewed in an access review."
```

author: "isabelleatmsft"
localization_priority: Normal

> ms.prod: "microsoft-identity-platform"

Reviewer: ```suggestion
ms.prod: "governance"
```

doc_type: resourcePageType
---

# accessReviewQueryScope resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
[!INCLUDE [accessreviews-disclaimer-v2](../../includes/accessreviews-disclaimer-v2.md)]

> accessReviewQueryScope defines what will be reviewed in an [accessReview](../resources/accessreviewsv2-root.md).

Reviewer: ```suggestion
An accessReviewQueryScope object defines what will be reviewed in an [accessReview](../resources/accessreviewsv2-root.md).
```

Inherits from [accessReviewScope](../resources/accessreviewscope.md).

## Properties
|Property|Type|Description|
|:---|:---|:---|
|query|String|The query representing what will be reviewed in an access review. Examples of this include /groups/{id}/members?$filter=…|

> |queryRoot|String|In the scenario where reviewers need to be specified dynamically, this property is used to indicate the relative source of the query. This property is only required if a relative query (like ./manager) is specified.|

Reviewer: ```suggestion
|queryRoot|String|In the scenario where reviewers need to be specified dynamically, this property is used to indicate the relative source of the query. This property is only required if a relative query is specified. For example, `./manager`.|
```
